safe_core/src/lib.rs

```diff
@@ -39,9 +39,9 @@ pub use ffi::*;
 
 // Export public core interface.
 
-pub use self::client::{
-    mdata_info, recoverable_apis, test_create_balance, AuthActions, Client, ClientKeys, MDataInfo,
-};
+#[cfg(any(test, feature = "testing"))]
```

Comment:
Same comment as above. We use this in real-network also. So we don't need it to be feature-gated.